mypyc/build.py

```diff
@@ -138,13 +138,64 @@ def get_mypy_config(paths: List[str],
 }}
 """
 
+# As far as I could tell, Windows lacks the rpath style features
+# we would need in order to do everything at load time, which means
+# that instead we get to do it dynamically.
```

Comment:
Explain what exactly we need to do at load time (need to find the shared dll somehow?).